mungegithub/submit-queue/.gitignore

```diff
@@ -1,3 +1,5 @@
 local.deployment.yaml
 local.secret.yaml
 local.service.yaml
+local.pvc.yaml
+local.pv.yaml
```

Comment:
there could be less copy-pasta amongst the various .gitignores if they're each `local.*.yaml`, unless the intent is to use this specificity as an extra safeguard